app/src/main/java/se/inera/intyg/minaintyg/config/WebSecurityConfig.java

```diff
@@ -140,6 +145,11 @@ public SecurityFilterChain filterChain(HttpSecurity http,
     return http.build();
   }
 
+  public AuthenticationFailureHandler getForwardAuthenticationFailureHandler() {
+    final var id = String.valueOf(UUID.randomUUID());
```

Comment:
Won't this create an id that will be used every time anyone fails to login. The randomise should be done on each failed attempt.

Also we need to do log the error so we can find the errorId in our server logs.

Maybe consider to create a custom failure handler that do these things:
1) generate errorId
2) log error using MonitoringLogService
3) redirect the user to /error/login/{errorId}

Reply:
You are correct, the previous implementation would only generate one errorId that would be used for all failures.

I updated the solution and instead used a CustomAuthenticationFailureHandler to handle failures, that generates an errorId, log & redirects the user to the correct endpoint.

This log is new, if we decide to use it, will we need to update indexes on elastic?